docs/en/source/installation/python-sdk.rst

```diff
@@ -1,113 +1,82 @@
-**********************************************
-Python SDK Quickstart
-**********************************************
+**********************************
+FuriosaAI Python SDK Installation
+**********************************
 
-Using the FuriosaAI NPU Python SDK library, you can easily write Python programs that utilize the NPU.
+FuriosaAI's NPU Python SDK provides a number of interfaces for accelerating models using the NPU,
+Python libraries, and command line tools.
 
+Minimum Requirements
+----------------------------------------
+* :doc:`./driver`
+* :doc:`./runtime`
+* Python 3.7 or later (refer to :any:`SetupPython` if necessary)
+* The latest version of pip (Upgrade to the latest version of pip with the following command)
 
-**Minimum Requirements**
-
-* :doc:`/installation/driver`
-* :doc:`/installation/runtime`  
-* Python 3.7 or higher (refer to :any:`SetupPython` if necessary)    
-  
-
-If the above requirements have been fulfilled, you can install the FuriosaAI NPU Python SDK library through PyPi.
-
-.. code-block:: sh
-
-  pip install --upgrade furiosa-sdk[runtime]~=0.1.0
+  .. code-block::
 
+        $ pip3 install --upgrade pip
 
-Running Simple Python Code
-=================================
+Installation
+----------------------------------------
 
-After the installation is complete, you can check whether the package is installed correctly by running the Python code as follows 
-and outputting the version information.
+The included Python library interface can be used with a number
+of command line tools and functions. The FuriosaAI Python SDK can be installed with PyPi as follows.
 
-.. code-block::
-  
-  >> from furiosa import runtime
-  INFO:furiosa_sdk_runtime._api.v1:successfully loaded dynamic library libnux.so.1.0.0
+.. code-block:: sh
 
-  >> runtime.__full_version__
-  'Furiosa SDK Runtime .release:0.1.1+907338a44e91f176495b3c24fce3d9b1e626a662 (libnux 0.3.0-dev 9418048e4 2021-03-29 02:59:26)'
+  # Install the FuriosaAI NPU Python SDK to use the Python interface, e.g. `import furiosa`
+  pip install --upgrade furiosa-sdk~=0.1.0
+  # Install additional tools, see below for details
+  pip install --upgrade furiosa-sdk[runtime,quantizer]~=0.1.0
+  # Install all additional tools
+  pip install --upgrade furiosa-sdk[full]~=0.1.0
 
+The following additional packages can be installed by using ``pip``. 
```

Comment:
I think it would be good to keep consistency when we use backticks.
If you want to use backticks with `pip`, please do the same with line 75.

If you want to go with backticks, I personally think a single backtick would be better. (e.g. \`pip\`).